Rubberduck.Parsing/VBA/RubberduckParser.cs

```diff
@@ -161,19 +159,55 @@ private void ParseAll()
                 _componentAttributes.Remove(invalidated);
             }
 
-            foreach (var vbComponent in modified)
+            foreach (var vbComponent in toParse)
             {
                 ParseAsync(vbComponent, CancellationToken.None);
             }
         }
 
+        private void AddBuiltInDeclarations(IReadOnlyList<VBProject> projects)
+        {
+            SyncComReferences(projects);
+
+            var finder = new DeclarationFinder(_state.AllDeclarations, new CommentNode[]{}, new IAnnotation[]{});
+            if (finder.MatchName(Tokens.Err).Any(item => item.IsBuiltIn 
+                && item.DeclarationType == DeclarationType.Variable 
+                && item.Accessibility == Accessibility.Global))
+            {
+                return;
+            }
+            
+            var vba = finder.FindProject("VBA");
+            Debug.Assert(vba != null);
+            
+            var errObject = finder.FindClass(vba, "ErrObject", true);
+            Debug.Assert(errObject != null);
+
+            var qualifiedName = new QualifiedModuleName(vba.IdentifierName, vba.IdentifierName, errObject.IdentifierName);
+            var err = new Declaration(new QualifiedMemberName(qualifiedName, Tokens.Err), vba, "Global", errObject.IdentifierName, true, false, Accessibility.Global, DeclarationType.Variable);
+            _state.AddDeclaration(err);
+
+            var debugClassName = new QualifiedModuleName(vba.IdentifierName, vba.IdentifierName, "DebugClass");
```

Comment:
@autoboosh I noticed - and that's a nice timing, because on my end I made the COM reflection code recursively resolve function return types, so the `Err` function is now known to return an `ErrObject` instance. The problem is that `Err.Raise` doesn't parse as a function call, so I cheated and added a global `Err` *object variable* - and it worked!

For some reason I can't seem to find the `Debug` class anywhere though.